web/src/core/interface/user.ts

```diff
@@ -0,0 +1,8 @@
+interface IUser {
```

Comment:
See how to create types from schemas [here](https://mongoosejs.com/docs/typescript/schemas.html) and implement something similar

Also don't add this `I` infront of any interface

```suggestion
interface User {
```